### PR TITLE
add instructions for rollup and vite+rollup

### PR DESCRIPTION
**Pull request by fipp**
added some docs about configuring "raw imports" for rollup and vite+rollup

### CODE REVIEW

File: src/howto/importing-ftl-files.md [hunk 1]
```diff
@@ -45,3 +45,39 @@ import enMessages from './en.ftl?raw'
...
+
+```js
+import { string } from 'rollup-plugin-string'
+
+export default {
+  ...
+  plugins: [
+    ...,
+    string({
+      include: /\.ftl$/i,
```
Comment (Demivan) [inline src/howto/importing-ftl-files.md:77]: Can this plugin check for `?raw` query string? This way you can continue using `?raw` without a need for `vite-raw-plugin`

Comment (fipp) [reply src/howto/importing-ftl-files.md:77]: I thought so too, I tried with regex `/\.ftl\?raw$/i` but i got the same error as before.

Comment (fipp) [reply src/howto/importing-ftl-files.md:77]: Just tried again and got the same behaviour ...

* With no rollup plugin and `?raw` import: `ENOENT: no such file or directory, open '/../src/i18n/nb_NO.ftl?raw'`
* with the plugin configured with the regex above, same error

**Comment by Demivan** [comment]
Thanks for the contribution

**Pull request merged**